source/includes/options-mongosqld.yaml

```
@@ -874,4 +874,22 @@ description: |
      +------+------+--------+-------+
 
 optional: true
+---
+program: mongosqld
+name: setParameter
+directive: option
+args: <polymorphic_type_conversion_mode | type_conversion_mode>
+description: |
+  .. list-table::
+   :header-rows: 1
+   * - Name
+     - Type
+     - Corresponds to
+   * - ``polymorphic_type_conversion_mode``
+     - string
+     - Determines how :binary:`~bin.mongosqld` evaluates document fields that are specified with multiple data types.
+   * - ``type_conversion_mode``
+     - string
+     - Specifies the semantics that :binary:`~bin.mongosqld` uses for type conversions, such as the CAST function.
```

```suggestion
     - Specifies which mode the :binary:`~bin.mongosqld` uses to
       :doc:`convert data types </reference/type-conversion>`.
       |bi-short| uses its own mode (``mongosql``) by default. Accepted
       values are ``mongosql`` and ``mysql``.
```
Wrap at 72 chars. Added link to data type conversion.